src/Fetcher/TululaFetcher.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Fetcher;
+
+use App\Entity\Conference;
+use App\Entity\Continent;
+use App\Entity\ExcludedTag;
+use App\Entity\Tag;
+use Doctrine\Persistence\ManagerRegistry;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\String\Slugger\AsciiSlugger;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TululaFetcher implements FetcherInterface
+{
+    const SOURCE = 'tulula';
+    const TAGS_SYNONYMS = [
+        'android',
+        'css',
+        'data',
+        'devops',
+        'dotnet',
+        'facebook',
+        'golang',
+        'html',
+        'ios',
+        'java',
+        'javascript',
+        'nodejs',
+        'php',
+        'python',
+        'react native',
+        'ruby',
+        'rust',
+        'security',
+        'ux',
+        'ai/ml',
+        'sql',
+        'iot',
+    ];
+
+    private $em;
+    private $logger;
+    private $tagRepository;
+    private $excludedTags;
+    private $locationGuesser;
+    private $slugger;
+    private HttpClientInterface $client;
+
+    public function __construct(ManagerRegistry $doctrine, LoggerInterface $logger, LocationGuesser $locationGuesser, HttpClientInterface $client)
+    {
+        $this->client = $client;
+        $this->em = $doctrine->getManager();
```

Comment:
You inject the MangerRegistry, but you only use it to get the EM. So inject the EM directly